Cargo.toml

```diff
@@ -26,6 +26,11 @@ zeroize = "1.5.7"
 bulletproofs = "4.0.0"
 curve25519-dalek-ng = "4.1.1"
 merlin = "3.0.0"
+aes = "0.8.1"
+ctr = "0.9.1"
+cbc = "0.1.2"
+cfb-mode = "0.8.1"
```

Comment:
I think we could retire cfb for now, and replace it with GCM which is the most popular mode for authenticated encryption (and for which there is a wycheproof test).
All in all, we can start with CTR, CBC, GCM for which we know of Sui applications already